caf: Add ble_state (#4180)
nrfconnect/sdk-nrf: applications/nrf_desktop/doc/dev_descr.rst

* Information about whether the peripheral supports the Low Latency Packet Mode (LLPM)
* Hardware ID (HW ID) of the peripheral

> In order to support the LLPM, the peripheral must use the SoftDevice Link Layer (:option:`CONFIG_BT_LL_SOFTDEVICE`) and :option:`CONFIG_CAF_BLE_USE_LLPM` Kconfig option must be enabled.

Reviewer: ```suggestion
To support the LLPM, the peripheral must use the SoftDevice Link Layer.
This means that you must enable both the :option:`CONFIG_BT_LL_SOFTDEVICE` and the :option:`CONFIG_CAF_BLE_USE_LLPM` Kconfig options.
```
`In order to` is just a fancy `to`.
